{{cookiecutter.project_name}}/scent.py

```diff
@@ -0,0 +1,40 @@
+{%- if cookiecutter.python_major_version == "2" -%}# -*- coding: utf-8 -*-
+
+{% endif -%}
+import os
+import subprocess
+
+from sniffer.api import select_runnable, file_validator, runnable
+from pync import Notifier
+
+
+watch_paths = ['{{cookiecutter.package_name}}/', 'tests/']
+
+
+@select_runnable('python_tests')
+@file_validator
+def py_files(filename):
+    return all((filename.endswith('.py'),
+               not os.path.basename(filename).startswith('.')))
+
+
+@runnable
+def python_tests(*args):
+
+    for count, (command, title) in enumerate((
+        (('make', 'test-unit'), "Unit Tests"),
+        (('make', 'test-int'), "Integration Tests"),
+        (('make', 'test-all'), "Combined Tests"),
+    ), start=1):
+
+        failure = subprocess.call(command)
+
+        if failure:
+            mark = "❌" * count
+            Notifier.notify(mark + " [FAIL] " + mark, title=title)
```

Comment:
add `group=1234` or some integer so each loop iteration doesn't create a separate notification and clutter up notification center. If the group is unique per test run, you will only get one notification per run. Using the epoch time is a good way to do this:

```patch
diff --git a/{{cookiecutter.project_name}}/scent.py b/{{cookiecutter.project_name}}/scent.py
index 6d7a84c..407acda 100644
--- a/{{cookiecutter.project_name}}/scent.py
+++ b/{{cookiecutter.project_name}}/scent.py
@@ -3,12 +3,13 @@
 {% endif -%}
 import os
 import subprocess
+import time

 from sniffer.api import select_runnable, file_validator, runnable
 from pync import Notifier


-watch_paths = ['{{cookiecutter.package_name}}/', 'tests/']
+watch_paths = ['demo/', 'tests/']


 @select_runnable('python_tests')
@@ -20,7 +21,7 @@ def py_files(filename):

 @runnable
 def python_tests(*args):
-
+    gid = int(time.time())
     for count, (command, title) in enumerate((
         (('make', 'test-unit'), "Unit Tests"),
         (('make', 'test-int'), "Integration Tests"),
@@ -31,10 +32,10 @@ def python_tests(*args):

         if failure:
             mark = "❌" * count
-            Notifier.notify(mark + " [FAIL] " + mark, title=title)
+            Notifier.notify(mark + " [FAIL] " + mark, title=title, group=gid)
             return False
         else:
             mark = "✅" * count
-            Notifier.notify(mark + " [PASS] " + mark, title=title)
+            Notifier.notify(mark + " [PASS] " + mark, title=title, group=gid)

     return True
```